core/lib/spree/testing_support/dummy_app.rb

```diff
@@ -41,6 +41,12 @@ def self.setup(gem_root:, lib_name:, auto_migrate: true)
     DummyApp::Application.config.root = root
     DummyApp::Application.initialize! unless DummyApp::Application.initialized?
 
+    # Raise on deprecation warnings.
+    # NOTE: This needs to happen after the appliaction is initilized.
```

Comment:
Just noticed a couple typos. Oops!

```suggestion
    # NOTE: This needs to happen after the application is initialized.

Reply:
Resolved.